src/cli/load-config.ts

```
@@ -2,6 +2,7 @@ import fs from 'node:fs/promises'
 import path from 'node:path'
 
 import { nodeTypes } from '@mdx-js/mdx'
+import { pluginSass } from '@rsbuild/plugin-sass'
```

ESLint

[error] Unable to resolve path to module '@rsbuild/plugin-sass'.

(import-x/no-unresolved)